code/index.html

```diff
@@ -20,25 +20,38 @@
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the character have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="pink">pink hair</option>
+          <option value="black">black hair</option>
+          <option value="grey">grey hair</option>
+          <option value="hidden">hidden hair</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eye color">
           <option value="hidden">hidden eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="black">black eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="green">green eyes</option>
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
+          <option value="hat">a hat</option>
           <!-- Add the other accessory option here. (hat)  -->
         </optgroup>
         <optgroup label="other">
           <option value="smoker">a smoking habit</option>
         </optgroup>
+        <optgroup label="look">
```

Comment:
Great that you've added some extra options and a new optgroup. Great way to practice and makes the game for fun :)